src/main/java/org/openstreetmap/atlas/geography/atlas/pbf/store/PbfMemoryStore.java

```diff
@@ -502,9 +503,25 @@ public boolean isAtlasPoint(final long identifier)
 
     public boolean isAtlasPoint(final Node node)
     {
-        // Each node will have a last edit time tag
-        return node.getTags().size() > AtlasTag.TAGS_FROM_OSM.size()
-                || containsNodeInRelations(node.getId());
+        if (containsNodeInRelations(node.getId()))
+        {
+            return true;
```

Comment:
So if an OSM node is part of a relation and it is at the end of an edge, then it would be have one node counterpart in Atlas and one point counterpart. Is that intentional?

Reply:
Plus, what is the reason we make an OSM node Atlas point because it is part of a relation?

Reply:
I am not sure why this was designed this way. I figured for this PR I would not touch that part of the logic, but you are correct this can be questionable... Created https://github.com/osmlab/atlas/issues/10 to track it.

Reply:
Cool. It can definitely be addressed later.

Reply:
I don't remember why we need this check, it doesn't seem correct. Agree to try to remove this later. Thanks!